docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -313,6 +313,67 @@ In the following example, a three-node cluster is created and uses a YCSB worklo
+
+{{< tip title="Tip" >}}
+
+To view the number of live tablets and the limits, open the **YB-Master UI** (`<master_host>:7000/`) and click the **Universe Summary** section. The total number of live tablet replicas is listed under "Active Tablet-Peers", and the limit is listed under "Tablet Peer Limit".
+{{< /tip >}}
+
+To disable the feature, set the flag `enforce_tablet_replica_limits` to false on all YB-Masters.
+
+{{<note title="YSQL restores are not fully supported">}}
+
+Currently, tablets created during a YSQL restore are not entirely covered by this feature. The YSQL restore flow creates tablets in two steps:
+
+1. The restored tables are created by executing SQL DDLs.
+1. If necessary, new tablets are created so the number of tablets supporting the restored tables matches the number of tablets supporting the backed up tables. Any tablets created during this step are _not_ checked against the cluster limits.
+
+{{</note>}}
+
+### Metrics
+
+The following table describes metrics related to tablet limits.
+
+| Metric | Description |
+| :----- | :---------- |
+| ts_supportable_tablet_peers | The number of tablet replicas this TServer can support. -1 if there is no limit. |
+| create_table_too_many_tablets | The number of create table requests failed because the cluster cannot support the additional tablet replicas. |
+| split_tablet_too_many_tablets | The number of split tablet operations failed because the cluster cannot support the additional tablet replicas. |
+
+### Example
+
+{{% explore-setup-single-local %}}
+
+Assuming a cluster has been properly configured, if you try to create a table beyond the configurable limit, the error message you can expect to see in a ysqlsh session is as follows:
+
+```sql
+create table foo (i int primary key, j int) split into 10 tablets;
+```
+
+```output
+ERROR:  Invalid table definition: Error creating table yugabyte.foo on the master: The requested number of tablet replicas (30) would cause the total running tablet replica count (102) to exceed the safe system maximum (93)
+```
+
+As a best practice, YugabyteDB has pre-computed sensible defaults for the memory limiting flags. By setting the [use_memory_defaults_optimized_for_ysql](../../../reference/configuration/yb-tserver/#use-memory-defaults-optimized-for-ysql) flag to true, the [tablet_overhead_size_percentage](../../../reference/configuration/yb-tserver/#tablet-overhead-size-percentage) is set with a sensible default as well. So, to use the memory limits, the only flag that needs to be set is [enforce_tablet_replica_limits](../../../reference/configuration/yb-master/#enforce-tablet-replica-limits).
+
+It is recommended to use the pre-computed memory defaults. Setting `use_memory_defaults_optimized_for_ysql` reserves memory for PostgreSQL, which is wasteful if not using YSQL. In this case, set the `tablet_overhead_size_percentage` flag to 10 on all YB-TServers.
```

Comment:
This is in the `Example` section currently. In my template it's in the `Best Practices` section. Is this change deliberate?

Reply:
I just added it as a conclusion for the topic.. but I see that you are viewing it as part of the example section, so ill update it accordingly.